content/information-distribution/source.md

```diff
@@ -15,18 +15,20 @@ event **must** be provided using the following model:
-| `sig_keys`          | `[string]`        |  x   | Public keys the source will use to cryptographically sign its articles. 🔧                                                           |
-| `website`           | `string`          |      | URL of the source's website, if there's one.                                                                                        |
-| `description`       | `localisedString` |      | Short description of the source and its publications.                                                                               |
-| `logo`              | `string`          |      | Logo of the source. If provided, must be a [`mxc://` URL](https://matrix.org/docs/spec/client_server/r0.4.0.html#id112).            |
-| `country`           | `string`          |      | Country of the source's owner. If provided, **must** be compliant with [ISO 3166](https://www.iso.org/iso-3166-country-codes.html). |
-| `custom`            | `object`          |      | Additional information for custom client implementations.                                                                           |
+| Parameter     | Type              | Req. | Description                                                                                                                                                |
+|:--------------|:------------------|:----:|:-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `name`        | `localisedString` |  x   | Name of the source.                                                                                                                                        |
+| `owner`       | `localisedString` |  x   | The company or individual maintaining this source.                                                                                                         |
+| `l10n`        | `lang`            |  x   | Languages of the source's publications.                                                                                                                    |
+| `sig_algo`    | `string`          |  x   | Algorithm the source will use to cryptographically sign its articles. 🔧                                                                                   |
+| `sig_keys`    | `[string]`        |  x   | Public keys the source will use to cryptographically sign its articles. 🔧                                                                                 |
+| `prev_id`     | `string`          |      | Matrix user ID of the Matrix user this source previously used to publish information. See [below](#change-of-matrix-user).                                 |
+| `prev_event`  | `string`          |      | ID of the latest event published by the source's previous user. Only valid if `prev_id` is set to a non-empty string. See [below](#change-of-matrix-user). |
```

Comment:
```suggestion
| `prev_event`  | `string`          |      | ID of the last legit event published by the source's previous user. Only valid if `prev_id` is set to a non-empty string. See [below](#change-of-matrix-user). |
```
There is the same change to do L161

Reply:
I'm not sure about the word "legit" which doesn't have an objective definition. Replacing with `the latest event published by the source with this user`.